README.md

```diff
@@ -162,3 +162,52 @@ container as a new Docker image (for example, by using *docker
 commit*), and then build the backend as described
 [above](#build-the-tensorflow-backend) with
 TRITON_TENSORFLOW_DOCKER_IMAGE set to refer to the new Docker image.
+
+
+## Using the Tensorflow Backend
+### Parameters
+
+Configuration of Tensorflow for a model is done through the Parameters section of the model's 'config.pbtxt' file. The parameters and their description are as follows.
+
+* `TF_NUM_INTRA_THREADS`: Number of threads to use for inference on the CPU. Should be a non-negative number.
+* `TF_NUM_INTER_THREADS`: Number of threads to use for inference on the CPU. Should be a non-negative number.
+* `TF_USE_PER_SESSION_THREADS`: Boolean value to see if per session thread is used. "Yes" is accepted as true
```

Comment:
With the rebase changes I am using "ParseBoolValue" from our backend repo. The TF_USE_PER_SESSION_THREADS should read: 
>  Boolean value to see if per session thread is used. "True", "On" and "1" are accepted as true.
We can also modify our backend utilis to treat "Yes" as  true and "No" as false. But for this release, Lets just change the documentation.